grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy

```diff
@@ -149,7 +149,7 @@ class Invoice implements Serializable {
         BigDecimal currentExchangeRate
         String defaultCurrencyCode = Holders.config.openboxes.locale.defaultCurrencyCode
         if (currencyUom?.code != defaultCurrencyCode) {
-            currentExchangeRate = UnitOfMeasureConversion.conversionRateLookup(defaultCurrencyCode, currencyUom?.code).list()
+            currentExchangeRate = UnitOfMeasureConversion.conversionRateLookup(defaultCurrencyCode, currencyUom?.code).get()
```

Comment:
Are we sure that we always will have one active conversion rate for a specific currency (I am not sure how `uniqueResult = true` already works there)? Perhaps it is worth improving the named query to sort by last updated date?